testing/jormungandr-testing-utils/src/testing/fragments/verifier.rs

```diff
@@ -181,6 +181,7 @@ impl FragmentVerifier {
         &self,
         duration: Duration,
         check: MemPoolCheck,
+        exit_on_pending: bool,
```

Comment:
There's an opaque boolean parameter that's one more than a good method signature should take.
Couple style suggestions:
- Turn it into an enum with descriptive variants.
- Maybe convert some of these parameters into setter methods on `FragmentVerifier`.